### PR TITLE
Port Scanner in Go

### LINKED ISSUE #705: PORT scanner in GOLANG
**Is your feature request related to a problem? Please describe.**
A Golang script that will scan  IP and PORT of a network

### PR DESCRIPTION
**Pull request by pritamp17**
# Description
Go-lang script for scanning specific or all open ports of the device.

Fixes #705

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

1new
2new
3new

### CODE REVIEW

**Comment by pritamp17** [comment]
@HarshCasper please do review this  PR.

File: Go/port-scanner-go/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (HarshCasper) [inline Go/port-scanner-go/readme.md:1]: Remove this extra line

File: Go/port-scanner-go/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Port scanner in Go
+Go-lang script for scanning specific or all open ports of the device.
```
Comment (HarshCasper) [inline Go/port-scanner-go/readme.md:3]: Add a specific use-case of Port Scanners for better understanding.

File: Go/port-scanner-go/readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# Packages Used
+1. fmt
+2. time
+3. sync
+4. strings
+5. net
```
Comment (HarshCasper) [inline Go/port-scanner-go/readme.md:10]: Use: 

```
-
-
```

instead of an ordered list

File: Go/port-scanner-go/readme.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+# How to run
+
+** Navigate to the project directory.
```
Comment (HarshCasper) [inline Go/port-scanner-go/readme.md:14]: Remove: `**`

File: Go/port-scanner-go/readme.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+go run port_scanner.go 
+```
+## Output
+
+![](https://i.imgur.com/xMil1Kh.jpg)
+![](https://i.imgur.com/VEQPbz3.jpg)
+![](https://i.imgur.com/tMiEYNk.jpg)
+
+## Author
+  Pritam Pawar.
```
Comment (HarshCasper) [inline Go/port-scanner-go/readme.md:29]: Add a hyperlink to your LinkedIn/Github

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Go/port-scanner-go/readme.md (hunk 5 above)
Comment (pritamp17) [reply Go/port-scanner-go/readme.md]: @HarshCasper I have made changes that you suggested

**Comment by HarshCasper** [comment]
Few changes more: 

- Rename `readme.md` to `README.md`
- Format your Code using [this](https://www.cleancss.com/go-beautify/) tool.

**Comment by pritamp17** [comment]
> Few changes more:
> 
> * Rename `readme.md` to `README.md`
> * Format your Code using [this](https://www.cleancss.com/go-beautify/) tool.

@HarshCasper I have made the suggested changes.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**